packages/kit/src/cli.js

```diff
@@ -158,28 +158,13 @@ prog
 		}
 	});
 
-// For the benefit of early-adopters. Can later be removed
+// TODO remove this after a few versions
 prog
 	.command('start')
-	.describe('Serve an already-built app')
-	.option('-p, --port', 'Port', 3000)
-	.option('-h, --host', 'Host (only use this on trusted networks)', 'localhost')
-	.option('-H, --https', 'Use self-signed HTTPS certificate', false)
-	.option('-o, --open', 'Open a browser tab', false)
```

Comment:
This shows the wrong error message (unknown flag vs the command has changed) to people using previously valid flags:

```sh
PS C:\Users\Jacob\Repositories\kit\examples\hn.svelte.dev> pnpx svelte-kit start
"svelte-kit preview" will now preview your production build locally. Note: it is not intended for production use
PS C:\Users\Jacob\Repositories\kit\examples\hn.svelte.dev> pnpx svelte-kit start --open

  ERROR
    Unknown option: --open

  Run `$ svelte-kit --help` for more info.
```

Reply:
ah, whoops 🙈 fixing, sorry